tests/integration/test_filter.py

```diff
@@ -142,3 +142,23 @@ def test_decompress_regular(tmpdir, httpbin):
         resp = urlopen(url).read()
         assert_cassette_has_one_response(cass)
     assert_is_json(resp)
+
+
+def test_before_record_request_corruption(tmpdir, httpbin):
+    """Modifying request in before_record_request should not effect outgoing request"""
```

Comment:
I believe "effect" should be "affect" with "a" to be correct?

Reply:
doh